pandas/core/series.py

```diff
@@ -3906,6 +3915,8 @@ def rename(self, index=None, **kwargs):
 
         Parameters
         ----------
+        axis : int or str
```

Comment:
This should probably be 
```
axis : {0 or ‘index’}
   Axis along which to fill missing values.
```
for consistency with Series methods that share docstrings and use the method in generic. see previous comment.